src/orion/algo/tpe.py

```diff
@@ -271,35 +300,35 @@ def _initialize(self):
             if shape and len(shape) != 1:
                 raise ValueError("TPE now only supports 1D shape.")
 
-    def seed_rng(self, seed):
+    def seed_rng(self, seed: int | Sequence[int] | None) -> None:
         """Seed the state of the random number generator.
 
         :param seed: Integer seed for the random number generator.
         """
+        super().seed_rng(seed)
```

Comment:
Why add super call?

Reply:
If the BaseAlgorithm ever adds a default implementation for seed_rng, then this will save us from having to debug flaky tests etc.
Until then, it's a no-op. Doesn't hurt to have it IMO.

Reply:
Alright, sounds good.